css/interface.css

```diff
@@ -425,7 +425,8 @@ header .btn,
 }
 
 .panel-title .fa {
-  cursor: pointer;
+  cursor: -webkit-grab;
+  cursor: -moz-grab;
```

Comment:
no `grab` ?

Reply:
Same on line `920`.

Reply:
Apparently not. I'm following the existing patterns, which seem to be working quite well.

Reply:
Weird, but ok